docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - .:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py migrate &&
+      python cinema/management/commands/wait_for_db.py &&
+      python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
```

Comment:
The `depends_on` option does not wait for `db` to be ready; it only ensures that the `db` container is started before the `cinema` container. Consider using a script or tool to wait for the database to be fully ready before starting the application.